src/origin_sdk/gql/queries/input_queries/utils.py

```diff
@@ -0,0 +1,52 @@
+from typing import Dict, Union
+
+RecursiveTree = Dict[str, Union["RecursiveTree", None, str, bool, list]]
+
+
+def get_new_line(level: int):
+    return "\n" + "\t" * level
+
+
+def tree_to_string(tree: RecursiveTree, level=1):
+    return (
+        (get_new_line(level)).join(
+            [
+                "{",
+                *[
+                    key
+                    if type(value) != dict
+                    else f"""{key} {
+                tree_to_string(value, level=level+1)
+            }"""
+                    for key, value in tree.items()
+                ],
+            ]
+        )
+        + f"{get_new_line(level-1)}}}"
+    )
+
+
+transform_fields: RecursiveTree = {"type": None, "value": None}
+
+transform_fields_with_cause: RecursiveTree = {
```

Comment:
This looks unused. I think I'd want to see it in variable_values_with_transforms

Reply:
I am copying the fragments we defined in the UI, I will double check I haven't missed something.